File: app/src/main/java/com/jetsynthesys/rightlife/apimodel/newreportfacescan/UserAnswers.java
package com.jetsynthesys.rightlife.apimodel.newreportfacescan;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class UserAnswers {
    @SerializedName("_id")
    public String id;
    public String userId;
    public String questionId;
    public String status;
    public String type;
    public List<Answer> answers;
    public List<Object> error; // Or a specific error class
    public String createdAt;
    public String updatedAt;
    @SerializedName("__v")
    public int v;
    public String subscriptionId;
}

File: app/src/main/java/com/jetsynthesys/rightlife/ui/healthcam/UserAnswers.kt
package com.jetsynthesys.rightlife.ui.healthcam

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class UserAnswers {
    @SerializedName("_id")
    @Expose
    var id: String? = null

    @SerializedName("userId")
    @Expose
    var userId: String? = null

    @SerializedName("questionId")
    @Expose
    var questionId: String? = null

    @SerializedName("status")
    @Expose
    var status: String? = null

    @SerializedName("type")
    @Expose
    var type: String? = null

    @SerializedName("answers")
    @Expose
    var answers: List<Answer>? = null

    @SerializedName("error")
    @Expose
    var error: List<Any>? = null

    @SerializedName("createdAt")
    @Expose
    var createdAt: String? = null

    @SerializedName("updatedAt")
    @Expose
    var updatedAt: String? = null

    @SerializedName("__v")
    @Expose
    var v: Int? = null

    @SerializedName("subscriptionId")
    @Expose
    var subscriptionId: String? = null
}